.gitignore

```diff
@@ -16,3 +16,4 @@
 
 Staticfile.auth
 coverage
+spec/tmp
```

Comment:
I would have put the file in `/tmp`, if a top-level dir existed. But at the moment, no govuk-developer-docs processes create that folder.

And as the test _relies_ on the directory existing, it would mean having to commit a `/tmp/.gitignore` that's visible forevermore on the landing page of the repo. I felt that as this is a quirk of a testing mechanism that may not be around forever, it's neater to just hide that smell in a `spec` subdirectory.

Reply:
We do have that elsewhere e.g. [here](https://github.com/alphagov/content-publisher/tree/master/tmp). I reckon: if we're going to have a smell, it may as well be as visible and unsurprising as possible.

Reply:
I've come around to the idea, now that I know Rails creates a `tmp/.keep` by default when initialising new projects 👍 thanks.